### PR TITLE
INTMDB-434 - Fixed private_endpoint output documentation

### LINKED ISSUE #713: Docs - Example - Return a Connection String - Azure Private Endpoint
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
C:\work\repo\orgs\novia-financial\cloud_infrastructure\projects\platform\c17\UK_South\3.5.mongodb.atlas>terraform -version
Terraform v1.1.7
on windows_amd64
+ provider registry.terraform.io/hashicorp/azurerm v3.1.0
+ provider registry.terraform.io/mongodb/mongodbatlas v1.3.1
```

### Terraform Configuration File 

In the document the example is given as
```hcl
output "plstring" {
    value = lookup(mongodbatlas_cluster.cluster-test.connection_strings[0].private_endpoint[0].srv_connection_string, azurerm_private_endpoint.test.id)
}
# Example return string: plstring = mongodb+srv://cluster-atlas-pl-0.za3fb.mongodb.net
```

### Steps to Reproduce

My code
```hcl
output "plstring" {
  value = lookup(mongodbatlas_cluster.main.connection_strings[0].private_endpoint[0].srv_connection_string, azurerm_private_endpoint.main.id)
}
```

### Expected Behavior
Should return the connection string

### Actual Behavior
```hcl
 Error: Invalid function argument
│
│   on output.tf line 28, in output "plstring":
│   28:   value = lookup(mongodbatlas_cluster.main.connection_strings[0].private_endpoint[0].srv_connection_string, azurerm_private_endpoint.main.id)
│     ├────────────────
│     │ mongodbatlas_cluster.main.connection_strings[0].private_endpoint[0].srv_connection_string is "mongodb+srv://redacted-mongodb-cluster-pl-0.3xib3.mongodb.net"
│
│ Invalid value for "inputMap" parameter: lookup() requires a map as the first argument.
```

### Other info
Both endpoints are showing up and connected
The terraform has been refreshed

### PR DESCRIPTION
**Pull request by evertsd**
## Description

Updated documentation on cluster connection_strings to contain instructions on how to output a private_endpoint srv_connection_string given an endpoint_id

Closes #713 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

**Review by themantissa — CHANGES_REQUESTED** [review]
Since we have GCP PSC as well now should we add that example too?

**Review by Zuhairahmed — CHANGES_REQUESTED** [review]
Can we move this to v1.6.1 release branch?

**Comment by evertsd** [comment]
@themantissa I changed the format a bit to be a generic example for getting a connection_string from a endpoint_service_id. 

The specific example given is for GCP, however it would work for Azure, AWS, or whatever other theoretical provider depending on how the endpoint_service_id is set. 

Then I've linked to full examples for AWS, Azure, and GCP.

**Review by matt-condon — APPROVED** [review]

File: examples/azure-atlas-privatelink/Readme.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ This project aims to provide an example of using Azure and MongoDB Atlas togethe
 ## Dependencies
 
 * Terraform v0.13
-* Microsoft Azure account 
-* A MongoDB Atlas account 
+* Microsoft Azure account
+* A MongoDB Atlas account
```
Comment (Zuhairahmed) [inline examples/azure-atlas-privatelink/Readme.md]:
```suggestion
* MongoDB Atlas account
```

File: examples/gcp-atlas-privatelink/Readme.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ This project aims to provide an example of setting up GCP Private Service Connec
 ## Dependencies
 
 * Terraform v0.13+
-* Google account 
-* A MongoDB Atlas account 
+* Google account
+* A MongoDB Atlas account
```
Comment (Zuhairahmed) [inline examples/gcp-atlas-privatelink/Readme.md]:
```suggestion
* MongoDB Atlas account
```

File: examples/gcp-atlas-privatelink/Readme.md [hunk 3]
```diff
@@ -6,13 +6,13 @@ This project aims to provide an example of setting up GCP Private Service Connec
 ## Dependencies
 
 * Terraform v0.13+
-* Google account 
-* A MongoDB Atlas account 
+* Google account
```
Comment (Zuhairahmed) [inline examples/gcp-atlas-privatelink/Readme.md]:
```suggestion
* Google Cloud account
```

**Review by Zuhairahmed — CHANGES_REQUESTED** [review]
Few minor edits and added a few formatting/typo commits directly. Also can you update Advanced Cluster docs as you did with Cluster to add the entire Return Connection String examples to help users? 

[Example - Return a Connection String](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/cluster#example---return-a-connection-string)

**Comment by evertsd** [comment]
Thanks for the feedback and suggestions @Zuhairahmed -- I've implemented and applied them.

**Review by Zuhairahmed — APPROVED** [review]
LGTM! Thanks @evertsd feel free to merge

**Pull request merged**